### PR TITLE
modify some names and comments in edmf

### PR DESCRIPTION
**Pull request by szy21**
## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/prognostic_equations/edmfx_closures.jl [hunk 1]
```diff
@@ -9,7 +9,7 @@ import ClimaCore.Fields as Fields
 """
     Return buoyancy on cell centers.
 """
-function ᶜbuoyancy(params, ᶜρ_ref::FT, ᶜρ::FT) where {FT}
+function ᶜphysical_buoyancy(params, ᶜρ_ref::FT, ᶜρ::FT) where {FT}
```
Comment (szy21) [inline src/prognostic_equations/edmfx_closures.jl:12]: @trontrytel I change the name to `ᶜphysical_buoyancy` as I find it a bit confusing that `ᶜbuoyancy` returns physical buoyancy, but `ᶠbuoyancy` returns covariant3 buoyancy

**Review by trontrytel — APPROVED** [review]

**Pull request merged**